Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:3.10.5 AS base
```

Comment:
Minor, but I'd be tempted to only specify as far as `python:3.10`; the reason being that the minor/patch version _shouldn't_ cause breaking changes, but might pick up security patches etc. so having it automatically update when the image does would be a positive